dataprofiler/tests/data_readers/test_base_data.py

```diff
@@ -93,7 +94,7 @@ def func1(self):
             data.test
 
         # validate it will take BaseData attribute over the data attribute
-        self.assertIsNone(data.data_type)
```

Comment:
why is this getting removed?

Reply:
In order to make `data_type` have type `str` instead of `Optional[str]`, I had to make it so it doesn't get set to anything in `BaseData` (and just has a type definition line for mypy). This makes it so you get an exception if you try getting the `data_type` in `BaseData` so @JGSweets recommended looking at another field to test the same behavior. In this case, I looked to make sure `options` is taken from `BaseData` instead of `FakeData` which would mean that `options` is an empty dict.

Reply:
correct